src/components/FilterDropdown.jsx

```diff
@@ -26,7 +26,7 @@ function FilterDropdown(props) {
     <FormControl variant="outlined" className={classes.formControl}>
       <InputLabel>{label}</InputLabel>
       <Select value={choice} onChange={selectChoice} label={label}>
-        {removeNone || (
```

Comment:
@gonzalo1330 I just learned from the test warning statements that using `||` is not really an option as it'll render `true`

Reply:
Ok yea I saw the change from main and include the updated change to this with my PR for sorting functionality